lib/rubocop/cop/lint/single_argument_dig.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Lint
+      # Sometimes using dig method of Hash class ends up with just a single
```

Comment:
Same goes for Arrays, btw - https://tiagoamaro.com.br/2016/08/27/ruby-2-3-dig/

Don't forget to add some tests that are not using literals, but rather variable names.